app-services/advanced-filter-convert-publish/main.go

```diff
@@ -31,6 +31,7 @@ const (
 )
 
 func main() {
+	os.Setenv("EDGEX_SECURITY_SECRET_STORE", "false")
```

Comment:
We should add a comment, on all these, that this is non-standard just for the example to run w/o user having to set this environment variable. Should not be use in real world code.....